de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/SarosComponent.java

```diff
@@ -94,6 +105,16 @@ public void projectOpened() {
 
   @Override
   public void projectClosed() {
-    // TODO: Update project
+    ISarosSessionManager sarosSessionManager =
```

Comment:
I guess this needs a lot more of explanation. If you look at the code you are just wondering why the lifecycle is not started in projectOpened and is not stopped/disposed on projectClosed

Reply:
This is just a WIP. It seems to work, but I wanted your opinion on what other pitfalls there are with this handling.

I was also wondering why we don't do anything in `projectOpened`. I will look into what the CTOR and what `projectOpened` should be used for.